Actions/Github-Helper.psm1

```diff
@@ -815,25 +815,49 @@ function Set-ContentLF {
 # }
 function Set-JsonContentLF {
     Param(
-        [parameter(mandatory = $true, ValueFromPipeline = $false)]
+        [parameter(Mandatory = $true, ValueFromPipeline = $false)]
         [string] $path,
-        [parameter(mandatory = $true, ValueFromPipeline = $true)]
+        [parameter(Mandatory = $true, ValueFromPipeline = $true)]
         [object] $object
     )
 
     Process {
-        $object | ConvertTo-Json -Depth 99 | Set-ContentLF -path $path
+        $object | ConvertTo-JsonLF | Set-ContentLF -path $path
+    }
+}
+
+ <#
+    .SYNOPSIS
+        Converts a JSON string to a JSON object with LF line endings.
+    .DESCRIPTION
+        Converts a JSON string to a JSON object with LF line endings.
+        This ensures the same formatting in different PowerShell versions.
+    .PARAMETER object
+        The JSON object to convert.
+    .EXAMPLE
+        $json = ConvertTo-JsonLF -object $myObject
+ #>
+function ConvertTo-JsonLF {
+    Param(
+        [parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [object] $object
+    )
+
+    Process {
+        $content = $object | ConvertTo-Json -Depth 99
         if ($PSVersionTable.PSVersion.Major -lt 6) {
             try {
-                $path = $ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath($path)
-                # This command will reformat a JSON file with LF line endings as PowerShell 7 would do it (when run using pwsh)
-                $command = "`$cr=[char]13;`$lf=[char]10;`$path='$path';`$content=Get-Content `$path -Encoding UTF8|ConvertFrom-Json|ConvertTo-Json -Depth 99;`$content=`$content -replace `$cr,'';`$content|Out-Host;[System.IO.File]::WriteAllText(`$path,`$content+`$lf)"
-                . pwsh -command $command
+                # This command will reformat a JSON content with LF line endings as PowerShell 7 would do it (when run using pwsh)
```

Comment:
This code returns the string with CR+LF - we need to remove the CR - else we will detect changes in settings file all the time.